prime-router/src/main/kotlin/history/azure/SubmissionFunction.kt

```diff
@@ -126,4 +131,42 @@ class SubmissionFunction(
     ): HttpResponseMessage {
         return this.getDetailedView(request, id)
     }
+
+    /**
+     *Endpoint for intermediary receivers to verify status of messages. It passes
```

Comment:
This comment is the same as the comment for `DeliveryFunction.getEtorMetadata`. Can we be more specific in each description so its easier to tell what the difference between the two is? For example, are both actually for "intermediary `receivers` to verify status of messages" or is one supposed to say `senders`?

Reply:
Updated to say `senders` in SubmissionFunction